.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: build
+
+on:
+  push:
+    branches: 
+      - "master"
+      - "issues-431-gh-actions" 
```

Comment:
Shouldn't this line be removed before merging?

Reply:
done